examples/osp-ubuntu-20.04.yaml

```diff
@@ -16,7 +16,7 @@ apiVersion: operatingsystemmanager.k8c.io/v1alpha1
 kind: OperatingSystemProfile
 metadata:
   name: osp-ubuntu-aws
-  namespace: kube-system
+  namespace: cloud-init-settings
```

Comment:
I understand why this change was made but please revert it. I think we should stick with `kube-system`.

Reply:
Not really, this change was made intentionally, please refer to the comment above.